### DISCUSSION TITLE
Lamp cleaner error when running in Docker

### DISCUSSION THREAD

**Discussion by superprogresor** [Sanster/IOPaint#197] [Q&A]
When starting in Docker, the following error appears "Error invoking remote method 'docker-start-container': Error: (HTTP code 500) server error - Mounts denied: The path /path/to/cache_dir is not shared from the host and is not known to Docker. You can configure shared paths from Docker -> Preferences... -> Resources -> File Sharing". I'm a newbie. How to fix this error?

**Comment by Sanster** [comment]
Please change ` /path/to/cache_dir` to the directory of your host machine, where you want to save the models
